wremnants/syst_tools.py

```diff
@@ -0,0 +1,21 @@
+import hist
+import numpy as np
+
+def scale_helicity_hist_to_variations(scale_hist):
+
+    s = hist.tag.Slicer()
+    # select nominal QCD scales, but keep the sliced axis at size 1 for broadcasting
+    nom_scale_hist = scale_hist[{"muRfact" : s[1.j:1.j+1], "muFfact" : s[1.j:1.j+1]}]
```

Comment:
I think the axis names aren't actually set correctly in the main script. Did you possibly test this only for the gen one?

Reply:
This has been reorganized now so that these axes are defined centrally (in theory_tools.py)

Reply:
Ok this was my mistake, was reading an old version of the file and the axes were mismatched